style.css

```diff
@@ -1 +1,172 @@
 /*  Add your styling here */
+body {
+  font: 1em sans-serif;
+  align-content: center;
+  align-items: center;
+}
+
+header {
+  width: 95%;
+  margin-left: auto;
+  margin-right: auto;
+  height: auto;
```

Comment:
These rules are repeated in all parts of your page. If you are writing the same CSS over and over, it's a sign you should create one CSS class and apply it to all those elements. For example you could name it `.section-wrapper` and apply it to the header, your 2 main sections and the footer.

Reply:
fixed it